frontend/model/notifications/nativeNotification.js

```diff
@@ -35,9 +45,13 @@ export const setupNativeNotificationsListeners = () => {
     typeof PushManager !== 'function' ||
     typeof ServiceWorker !== 'function' ||
     typeof navigator.serviceWorker !== 'object'
-  ) return
+  ) {
+    console.warn("Notifications aren't available in this browser!")
+    return
+  }
 
   if (
+    !navigator.vendor === 'Apple Computer, Inc.' && // WebKit doesn't work
```

Comment:
1. This check is always false, causing the `if` block to be ignored. The bug is that it should be `navigator.vendor !== 'Apple Computer, Inc.'` (or alternatively, `!(navigator.vendor === 'Apple Computer, Inc.')`).
2. Not a fan of UA sniffing like this, but in this scenario it might be the only option. However, have you been able to check if this `onchange` test works on macOS using different browsers? Apple cited OS limitations for this not working.

Reply:
Thanks, pushed a fix. And yes I tested 3 browsers on macOS (Brave, FF, and Safari). Both Brave and FF support the change listeners (although FF only notifies of changes to `'notifications'`).